src/polyfills.js

```diff
@@ -16,8 +16,10 @@
 
 // Importing the document-register-element module has the side effect
 // of installing the custom elements polyfill if necessary.
+/*eslint-disable */
```

Comment:
this should no longer be needed. we now allow imports > 80 chars

Reply:
Still needed or is this very new?

Reply:
@cramforce on rebase you shouldn't need to do the disable / enable dance